src/tcgwars/logic/impl/gen3/LegendMaker.groovy

```diff
@@ -393,6 +401,7 @@ public enum LegendMaker implements LogicCardInfo {
           }
           onAttack {
             attachEnergyFrom(type : F, my.discard, self)
+            //TODO: Confirm the energy being attached maybe?
```

Comment:
Yeah, for attacks/abilities that can copy this attack, should probably put the `attackRequirement` in an if as well and contain these in it. At least from my understanding from what @ufodynasty has said.

Reply:
You mean as in duplicating it? Having both the assert and do an if? Just to confirm.

Reply:
Gonna leave the TODO in here, as I still plan on working out a check for "energy got attached" check at some point. Added the double conditional back though.